docs/sources/tempo/release-notes/v2-7.md

```diff
@@ -0,0 +1,248 @@
+
+Most Tempo installations use the receivers with the default configuration:
+
+```yaml
+distributor:
+  receivers:
+    otlp:
+      protocols:
+        grpc:
+        http:
+```
+
+This used to work fine since the receivers defaulted to `0.0.0.0:4317` and `0.0.0.0:4318` respectively. With the changes to replace unspecified addresses, the receivers now default to `localhost:4317` and `localhost:4318`.
+
+As a result, connections to Tempo running in a Docker container won't work anymore.
+
+To workaround this, you need to specify the address you want to bind to explicitly. For instance, if Tempo is running in a container with hostname `tempo`, this should work:
+
+```yaml
+# ...
+        http:
+          endpoint: "tempo:4318"
+```
+
+You can also explicitly bind to `0.0.0.0` still, but this has potential security risks:
+
+```yaml
+# ...
+        http:
+          endpoint: "0.0.0.0:4318"
+```
+
+### Maximum spans per span set
+
+A new `max_spans_per_span_set` limit is enabled by default and set to 100.
+Set it to `0` to restore the old behavior (unlimited).
+Otherwise, spans beyond the configured max are dropped. ([#4275](https://github.com/grafana/tempo/pull/4383))
+
+```
+query_frontend:
+  search:
+      max_spans_per_span_set: 0
+```
+
+### Tempo serverless deprecation
+
+Tempo serverless is now officially deprecated and will be removed in an upcoming release.
+Prepare to migrate any serverless workflows to alternative deployments. ([#4017](https://github.com/grafana/tempo/pull/4017), [documentation](https://grafana.com/docs/tempo/latest/operations/backend_search/#serverless-environment))
+
+There are no changes to this release for serverless. However, you’ll need to remove these configurations before the next release.
+
+### Anchored regular expressioon matchers in TraceQL
+
+TraceQL now uses the Prometheus “fast regex” engine to accelerate regular expression-based filtering ([#4329](https://github.com/grafana/tempo/pull/4329)).
+As part of this update, all regex matches are now fully anchored.
+This breaking change means `span.foo =~ "bar"` is evaluated as `span.foo =~ "^bar$"`.
+Update any affected queries accordingly.
+
+For more information, refer to the [Comparison operators TraceQL](http://localhost:3002/docs/tempo/<TEMPO_VERSION>/traceql/#comparison-operators) documentation.
+
+### Migration from OpenTracing to OpenTelemetry
+
+The `use_otel_tracer` option is removed.
+Configure your spans via standard OpenTelemetry environment variables.
+For Jaeger exporting, set `OTEL_TRACES_EXPORTER=jaeger`.For more information, refer to the [OpenTelemetry documentation](https://www.google.com/url?q=https://opentelemetry.io/docs/languages/sdk-configuration/&sa=D&source=docs&ust=1736460391410238&usg=AOvVaw3bykVWwn34XfhrnFK73uM_). ([#3646](https://github.com/grafana/tempo/pull/3646))
+
+### Deprecation
```

Comment:
already mentioned above in "Tempo serverless deprecation"?

Reply:
I am so glad you're reviewing this.

Reply:
Fixed.